keyboards/dz60/dz60.c

```diff
@@ -26,3 +26,13 @@ void led_set_kb(uint8_t usb_led) {
   // Set this per keymap, instead of globally
   led_set_user(usb_led);
 }
+
+void led_set_user(uint8_t usb_led) {
```

Comment:
This block really should be in `led_set_kb`

Reply:
got it!